roles/kubernetes/tasks/main.yml

```diff
@@ -6,7 +6,8 @@
 
 - name: Add kubernetes apt repo
   apt_repository:
-    repo: "deb http://apt.kubernetes.io/ kubernetes-{{ ansible_distribution_release|lower }} main"
+    # no repo available for bionic, pinning to xenial
+    repo: "deb http://apt.kubernetes.io/ kubernetes-xenial main"
```

Comment:
I think we could just download a binary directly but I want to keep it as similar to the previous version until we can get off this build pipeline